pkg/protocol/engine/eviction/state_test.go

```diff
@@ -17,7 +17,7 @@ func TestState_RootBlocks(t *testing.T) {
 	prunableStorage := prunable.New(database.Config{
 		Engine:    hivedb.EngineMapDB,
 		Directory: t.TempDir(),
-	}, errorHandler)
+	}, 0, errorHandler)
```

Comment:
We test somewhere for the rootCommitment right? It is kinda tricky and it is worth to make sure.

Reply:
We do in the startup test: https://github.com/iotaledger/iota-core/blob/958554fb848c1f0b4c131186a43cd9a4281cad30/pkg/tests/protocol_startup_test.go#L451